ITC205_BaseLine_Code/src/Main.java

```diff
@@ -1,5 +1,7 @@
 import java.text.SimpleDateFormat;
 import java.util.Scanner;
+//Ready for static review
```

Comment:
D18 – add extra blank line between methods – line 187 and 192

Reply:
D18:  Amended extra white line spacing

Reply:
Confirmed Fixed - Facilitator